Cargo.toml

```diff
@@ -37,3 +42,8 @@ lazy_static = "1.4"
 
 [target.'cfg(fuzzing_debug)'.dependencies]
 memmap = "0.7"
+
+[[bin]]
+name = "cargo-hfuzz"
+# requires the `cli` feature, but if it is forgotten, we want to fail compilation instead of silently skipping this binary
+# we use conditionnal compilation to throw the error
```

Comment:
Not sure I would want to go this way, I'd prefer to split it into two crates and avoid the end user chores.

Reply:
IDK, maintaining two crates, their synchronization, and making their doc reference each other in a non confusing way seems difficult too.

It would make sense to name the binary crate `hfuzz` though..

But the feature flag is not so bad, the user will try `cargo install honggfuzz`, will get a clear error stating that the flag is missing, and try again with `cargo install honggfuzz --features=cli`.

In the docs, the later command will be directly shown.